test/util/IdTableHelpers.h

```diff
@@ -115,24 +112,8 @@ void compareIdTableWithExpectedContent(
 
   // Sort both the table and the expectedContent, so that both have a definite
   // form for comparison.
-  // TODO Instead of this, we could use std::ranges::lexicographical_compare
-  // for the body of the lambda, but that is currently not possible, because
-  // the rows of an IdTable are not iterable.
-  auto sortFunction = [](const auto& row1, const auto& row2) {
-    size_t i{0};
-    while (i < (row1.numColumns() - 1) && row1[i] == row2[i]) {
-      i++;
-    }
-    return row1[i] < row2[i];
-  };
-  /*
-   * TODO Introduce functionality to the IdTable-class, so that
-   * std::ranges::sort can be used instead of std::sort. Currently it seems like
-   * the iterators , produced by IdTable, aren't the right type.
-   */
-  std::sort(localTable.begin(), localTable.end(), sortFunction);
-  std::sort(localExpectedContent.begin(), localExpectedContent.end(),
-            sortFunction);
+  std::ranges::sort(localTable, std::ranges::lexicographical_compare);
+  std::ranges::sort(localExpectedContent, std::ranges::lexicographical_compare);
```

Comment:
Nothing to do here for now, but funny thing:
I first thought that this shouldn't work (that you would have to wrap `lexicographical_compare` into a lambda),
but since it is a Niebloid, this works (by the way that the niebloid is implemented....). You still do not have to know, what a niebloid is, but this consequence is nice:)